package.json

```diff
@@ -31,7 +31,7 @@
     "rollup-plugin-prettier": "^0.4.0",
     "rollup-plugin-uglify": "^3.0.0",
     "sinon": "^7.3.1",
-    "webpack": "^4.10.2"
+    "webpack": "^2.7.0"
```

Comment:
I'm pretty sure this is _not_ a change we want to introduce. Can we roll back this change and the (I'm assuming) associated change to package-lock.json?